ui/js/component/modalIncompatibleDaemon/view.jsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import { Modal } from "component/modal";
+
+class ModalIncompatibleDaemon extends React.PureComponent {
+  render() {
+    const { quit, skipWrongDaemonNotice } = this.props;
+
+    return (
+      <Modal
+        isOpen={true}
+        contentLabel={__("Incompatible daemon running")}
+        type="confirm"
+        confirmButtonLabel={__("Exit")}
+        abortButtonLabel={__("Continue")}
+        onConfirmed={quit}
+        onAborted={skipWrongDaemonNotice}
+      >
+        {__(
+          "An incompatible version of the LBRY daemon is running. If you continue to receive this message after starting LBRY again, try restarting your computer."
```

Comment:
"This browser is running with an incompatible version of the LBRY protocol and your install must be repaired."

[Quit and Learn More]

.
.
.
.

(or if that's bad UX, maybe:)

[Continue]

&lt;meta&gt;Clicking continue will close the app and provide further instructions.&lt;/meta&gt;